README.md

```diff
@@ -1,20 +1,38 @@
-# nRF Connect Boilerplate
+# nRF Connect Board Configurator
 
 [![Build Status](https://dev.azure.com/NordicSemiconductor/Wayland/_apis/build/status/pc-nrfconnect-boilerplate?branchName=master)](https://dev.azure.com/NordicSemiconductor/Wayland/_build/latest?definitionId=10&branchName=master)
 [![License](https://img.shields.io/badge/license-Modified%20BSD%20License-blue.svg)](LICENSE)
 
-_nRF Connect Boilerplate_ provides a starting point for developing apps that can
-be launched by
-[nRF Connect for Desktop](https://github.com/NordicSemiconductor/pc-nrfconnect-launcher).
+nRF Connect Board Configurator is an application that you can use to update the
+configuration of the board controller on Nordic Development kits. The board
+controller is the firmware running on the Interface MCU that controls the
+behavior of the DK.
+
+**Note:** The current Board Configurator version is experimental. This means
+that the application is incomplete in functionality and verification and will
```

Comment:
W: I don't quite understand what "incomplete in verification" means, might be unclear for other people as well.